cmd/flags.go

```diff
@@ -328,6 +351,21 @@ func ValidateNoArgs(ctx *cli.Context) error {
 	return nil
 }
 
+func ValidateNetworkFlags(ctx *cli.Context) error {
+
+	networkFlagsCount := 0
+	for _, flag := range NetworkFlags {
+		if ctx.IsSet(flag.Names()[0]) {
+			networkFlagsCount++
+		}
+	}
+
+	if networkFlagsCount > 1 {
+		return fmt.Errorf("cannot use more than one network flag at the same time")
```

Comment:
No need to use `Errorf` since there is no variable in the error message.

However, it is not obvious to the user to know what "network flag" mean.
A good idea would be to list in the error message all possible network flags and indicate to the user they can use only one of them.

Reply:
alright, take a look at changes